rllib/evaluation/rollout_worker.py

```diff
@@ -1457,7 +1457,7 @@ def get_filters(self, flush_after: bool = False) -> Dict:
         return return_filters
 
     @DeveloperAPI
-    def save(self) -> bytes:
+    def get_state(self) -> bytes:
```

Comment:
Unify naming:
`get_state` -> returns dict that completely describes the current state of the object
`set_state(state)` -> can be used to restore to the state (which may have been returned from a get_state call)
`save` -> creates checkpoint on disk and returns path where stuff was saved